iModelCore/ecobjects/src/ECInstance.cpp

```diff
@@ -2687,6 +2698,12 @@ struct  InstanceXmlReader
                 return InstanceReadStatus::ECSchemaNotFound;
                 }
 
+            if (ECObjectsStatus::PruneSchema == schemaStatus)
+                {
+                LOG.infov("Skipping finding of the class '%s' because its schema '%s' is being pruned.", m_className.c_str(), m_fullSchemaName.c_str());
+                return InstanceReadStatus::ECSchemaNotFound;

```

Comment:
Can this be made debug or trace instead?

Reply:
Changed logging level to debug